src/components/ReportActionItem/ReportPreview.js

```diff
@@ -123,7 +141,7 @@ function ReportPreview(props) {
                             )}
                         </View>
                     </View>
-                    {isCurrentUserManager && !ReportUtils.isSettled(props.iouReport.reportID) && (
+                    {props.iouReport && isCurrentUserManager && !ReportUtils.isSettled(props.iouReport.reportID) && !props.iouReport.isWaitingOnBankAccount && (
```

Comment:
We have `{}` as a default value for the iouReport prop in line 83, so `props.iouReport` is always truthy

```suggestion
                    {isCurrentUserManager && !ReportUtils.isSettled(props.iouReport.reportID) && !props.iouReport.isWaitingOnBankAccount && (
```